packages/block-editor/src/layouts/flex.js

```diff
@@ -18,11 +19,18 @@ export default {
 	},
 
 	save: function FlexLayoutStyle( { selector } ) {
+		const blockGapSupport = useSetting( 'spacing.blockGap' );
+		const hasBlockGapStylesSupport = blockGapSupport !== null;
+
 		return (
 			<style>{ `
 				${ appendSelectors( selector ) } {
 					display: flex;
-					gap: var( --wp--style--block-gap, 0.5em );
+					gap: ${
```

Comment:
Just double-checking my understanding of the layout support — this saves the `style` markup to the block's post content? Following along, I imagine we don't have to worry about deprecations with this change, because it's unlikely anyone has set the `spacing.blockGap` to `null` on live sites already? It looks like in all other cases the existing markup will be the same, so there won't be an issue with block markup validation 👍

Reply:
this particular change is about the "flex" layout which is actually used anywhere live yet. Basically when the theme don't allow styles for blockGap, I just used a fixed one.

Reply:
Sounds good, thanks for clarifying!